### DISCUSSION TITLE
How much power can go true J4?

### DISCUSSION THREAD

**Discussion by gabrieleisele** [PiSupply/PiJuice#736] [Q&A]
How much power can go true J4. I want to use J4 to connect a 10 V 2A  (=20 W) power supply to both charge Pijuice and and run the PI 4.  Is that possible?

**Comment by tvoverbeek** [comment]
input is specified as 4.2 - 10V.
At 10V the PiJucie will only draw what it needs to charge the battery and power the Pi, which could certainly be less than 2A.
Should be nor problem for you as long as the delivered voltage is 10V.
